client/my-sites/sidebar/index.jsx

```diff
@@ -876,10 +877,16 @@ export class MySitesSidebar extends Component {
 					</ul>
 				</SidebarMenu>
 
-				{ isEnabled( 'jetpack/features-section' ) && this.jetpack() }
+				{
+					// Since we offer a path to trigger an Automated Transfer for Business plan
+					// owners, we need to verify if the current site is eligible for that.
+					isEnabled( 'jetpack/features-section' ) && (
+						<QueryEligibility siteId={ this.props.siteId } />
```

Comment:
Why does the code query the eligibility in the sidebar instead of on the upsell page?

Reply:
My logic maybe flawed, but I think we need to query the eligibility before going to the AT page since it's one of the conditions we check for. If we move the query to the upsell page it may not work for all users. I can imagine non-AT Business users non going through the upsell page (because they already have a Business plan) and the skipping the Automated Transfer page because we think they aren't eligible for AT (the query component was never rendered).

There are probably other better places to put the query component that I'm not aware of, so please let me know if you think I should move it to somewhere else. In the end, we want to be sure that we query the eligibility before going through that controller.

Reply:
@rcanepa - We can defer the output of the page and show a loading icon, similar to what we do for the `[UpsellSwitch](https://github.com/Automattic/wp-calypso/blob/a3e4e4da751749759d639296819dac2ffa0b61b5/client/components/jetpack/upsell-switch/index.tsx#L13-L13)` component.

I feel that querying eligibility in the sidebar is an extraneous API call and invites a race condition. I'd be happy to branch off the PR with a similar approach to what I linked above if that'd help?

Reply:
Yes! I like that approach a lot and I totally agree that the other can be affected by a race condition. 

> I'd be happy to branch off the PR with a similar approach to what I linked above if that'd help?

Sure. Feel free to use this one as well if you want to. I don't want to block other tasks because of this one.